editor.planx.uk/src/pages/FlowEditor/components/Flow/components/Checklist.tsx

```diff
@@ -92,9 +94,18 @@ const Checklist: React.FC<Props> = React.memo((props) => {
             onContextMenu={handleContext}
             ref={drag}
           >
+            {props.data?.img && (
+              <Thumbnail
+                imageSource={props.data?.img}
+                imageAltText={props.data?.text}
+              />
+            )}
             {Icon && <Icon />}
             <span>{props.text}</span>
           </Link>
+          {props.data?.fn && (
+            <DataField value={props.data.fn} variant="parent" />
+          )}
```

Comment:
Small styling bug outstanding here actually - we need to get all attributes wrapped in the `Link` I think in order to avoid disjointed "visited" green border styling - navigating a mix of scss classnames & React component styles here though now which is a bit tricky! 
![Screenshot from 2024-10-10 11-13-00](https://github.com/user-attachments/assets/758e0685-f0e3-4558-8550-9ca778a41218)

Reply:
Updating line 113 of editor.planx.uk/src/pages/FlowEditor/floweditor.scss to this should do the trick:
`&.wasVisited > div a,
  &.wasVisited > a {`

Reply:
Thanks! Rebasing to that change fixed borders :heavy_plus_sign: